composer.json

```diff
@@ -23,6 +23,7 @@
     "require": {
         "php": "^7.1",
         "cocur/slugify": "^2.0 || ^3.0",
+        "doctrine/mongodb-odm": "^2.0",
```

Comment:
This is an optional dependency. So please add it to `require-dev` and add a `conflict` entry for unsupported versions